packages/node_modules/@webex/internal-plugin-conversation/test/unit/spec/share-activity.js

```diff
@@ -111,5 +112,34 @@ describe('plugin-conversation', () => {
         assert.isTrue(spy.calledWith(inputData));
       });
     });
+
+    describe('#addGif', () => {
+      const fakeHappyGif = {name: 'happy gif.gif'};
+      const fakeSadGif = {name: 'sad gif.gif'};
```

Comment:
Can you make this one `sad-gif`
I'd like one to have a space and one to be cased